### PR TITLE
Symbols UI Fixes

### PR DESCRIPTION
**Pull request by shishirx34**
Addresses #6559 

UI changes:

.nupkg vs .snupkg
image

warning and metadata:
image

delete default to dropdown:
image

### CODE REVIEW

**Comment by shishirx34** [comment]
@diverdan92 - please review the UI and strings.

**Comment by diverdan92** [comment]
@shishirx34 - 
Warning and metadata:
> There is an existing symbols package (.snupkg) for the below package ID and version. Submitting this request will replace the previously uploaded symbols package as well as the corresponding symbol files from the symbol server.
Delete:
> Deleting this symbols package will remove all symbols in this package from the symbol server and make them unavailable. 

Otherwise, LGTM.

File: src/NuGetGallery.Core/Extensions/PackageExtensions.cs [hunk 1]
```diff
@@ -19,5 +19,16 @@ public static SymbolPackage LatestSymbolPackage(this Package package)
...
         }
+
+        /// <summary>
+        /// Returns true if there exists a symbol package which is latest and is in
+        /// available state, false otherwise.
+        /// </summary>
+        public static bool IsLatestSymbolPackageAvailable(this Package package)
+        {
+            var latestSymbolPackage = package.LatestSymbolPackage();
+            return latestSymbolPackage != null
```
Comment (cristinamanum) [inline src/NuGetGallery.Core/Extensions/PackageExtensions.cs]: Why it needs to be the latest and not getting the available package if any?

Comment (shishirx34) [reply src/NuGetGallery.Core/Extensions/PackageExtensions.cs]: Created a tracking issue: https://github.com/NuGet/Engineering/issues/1797

**Pull request merged**